Assignment7&8/Assignment/Assignment/SampleData.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Assignment
+{
+    public class SampleData : ISampleData
+    {
+        // 1.
+        public IEnumerable<string> CsvRows => File.ReadAllLines("People.csv").Skip(1);
+
+        // 2.
+        public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows()
+        {
+            List<string> uniqueSortedStates = new();
+            foreach (var row in CsvRows)
+            {
+                var splitRow = row.Split(",");
+                if (!uniqueSortedStates.Contains(splitRow[6]))
+                {
+                    uniqueSortedStates.Add(splitRow[6]);
+                }
+            }
+            uniqueSortedStates.Sort();
+            return uniqueSortedStates;
+        }
+
+        // 3.
+        public string GetAggregateSortedListOfStatesUsingCsvRows()
+        {
+            var uniqueSortedStatesArray = GetUniqueSortedListOfStatesGivenCsvRows().ToArray();
+            var uniqueSortedStates = string.Join(",", uniqueSortedStatesArray);
+            return uniqueSortedStates;
+        }
```

Comment:
```suggestion
        => string.Join(",", GetUniqueSortedListOfStatesGivenCsvRows());
```
You could consider making this less verbose